src/components/labels/Label.jsx

```diff
@@ -0,0 +1,183 @@
+// @flow strict
+
+import React from 'react';
+import type {Node} from 'react';
+import classNames from 'classnames';
+import Text from '../text/Text';
+import * as IconModule from '../icons/Icon';
```

Comment:
import Icon, {TYPE: ICON_TYPE} from '../icons/Icon';`?

Reply:
🧐

Reply:
@matzimowski This import would not work I think, there is no export of `ICON_TYPE`. I need to also use `IconTypeType` which I get from `IconModule `. I saw we do it this way in `IconAsButton`. But maybe there is a better way...

Reply:
ok, let's stick to what we have. 👌